### PR TITLE
Add license notice to primary init file

### LINKED ISSUE #57: check license file
`LICENSE.md` contains instruction after terms and conditions. This can be removed and the relevant information can be copied somewhere else instead.

### PR DESCRIPTION
**Pull request by michaelosthege**
Closes #57

### CODE REVIEW

**Review by lhelleckes — COMMENTED** [review]
Ideally, you should add an email or some other form of contact as well. Maybe Marco?

File: bletl/__init__.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (michaelosthege) [inline bletl/__init__.py:15]:
```suggestion
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
# For more information contact the maintainers of https://github.com/JuBiotech.
```
like this?

**Comment by michaelosthege** [comment]
- [x] The JuBiotech maintainers know my real name.

At least one of the following two applies:

- [x] The JuBiotech maintainers know my current employer.

- [ ] I am *not* making this contribution on behalf of my current employer.

**Comment by michaelosthege** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Review by lhelleckes — APPROVED** [review]

**Pull request merged**